src/AnalyticsConfig.ts

```diff
@@ -0,0 +1,23 @@
+import {Environment} from './models';
+import {Region} from './models';
+
+/**
+ * The main configuration options for Analytics Events.
+ *
+ * @public
+ */
+export interface AnalyticsConfig {
+    key?: string;
```

Comment:
Lets add a comment explaining that only one or other needs to be set

Reply:
Added comment